docs/src/main/asciidoc/_elytron/migrate/Caching_Migration.adoc

```diff
@@ -83,7 +83,7 @@ These can then be used in a security domain and subsequently an authentication f
 ./subsystem=elytron/http-authentication-factory=application-security-http:add(http-server-mechanism-factory=global, security-domain=application-security, mechanism-configurations=[{mechanism-name=BASIC}])
 ----
 
-In this final step it is very important that the caching-realm is referenced rather than the original realm otherwise caching will be bypassed.
+In this final step it is very important that the caching-realm is referenced rather than the original realm otherwise caching will be bypassed. When credentials are updated externally in the LDAP server, the obsolete user's credential will be removed when the user successfully authenticates with the updated credential.
```

Comment:
s/the obsolete user's credential/the user's obsolete credential

Reply:
@Skyllarr I think this change still needs to be made.

Reply:
@fjuma Thank you, fixed!